Show columns clarifications (#1826)
neo4j/docs-operations: modules/ROOT/pages/authentication-authorization/manage-roles.adoc

> Available roles can be seen using `SHOW ROLES`.

Reviewer: ```suggestion
You can view all available roles using the Cypher command `SHOW ROLES`.
```

> This returns a single column.

Reviewer: ```suggestion
```

> [options="header", width="100%", cols="2a,4,2m"]

Reviewer: ```suggestion
.`SHOW ROLES` output
[options="header", width="100%", cols="2a,4,2m"]
```

There are multiple versions of this command, the default being `SHOW ALL ROLES`.
To only show roles that are assigned to users, the command is `SHOW POPULATED ROLES`.
To see which users are assigned to which roles, `WITH USERS` can be added to the command.
The command produces a row per role per user and yields the following column in addition to the one output by `SHOW ROLES`:

> [options="header", width="100%", cols="2a,4,2m"]

Reviewer: ```suggestion
.`SHOW POPULATED ROLES WITH USERS` output
[options="header", width="100%", cols="2a,4,2m"]
```

|===
| Column
| Description
| Type

| member
| User name
| STRING
|===

> Since this gives a result with one row for each user, if a role is assigned to two users it will show up twice.

Reviewer: ```suggestion
Since this gives a result with one row for each user, it shows up twice if a role is assigned to two users.
```
